amxmodx/modules.cpp

```diff
@@ -187,7 +187,7 @@ int load_amxscript_internal(AMX *amx, void **program, const char *filename, char
 	bool will_be_debugged = false;
 	tagAMX_DBG *pDbg = NULL;
 
-	if ((int)CVAR_GET_FLOAT("amx_debug") >= 2 || debug)
+	if (amxmodx_debug->value >= 2.0f || debug)
```

Comment:
Could you please elaborate why you changed this?

Reply:
@WPMGPRoSToTeMa ->value is floating point , and it executed faster? ( I created cvar for using in amx_ExecPerf and this cvar can be used here, directly without execute CVAR_GET_FLOAT )

Reply:
Bypassing the string lookup is fine, but it should be `(int)amxmodx_debug->value == 2`. The value is an integer so using fuzzy comparisons is confusing.

Reply:
Okay I do it. 
This is can't decrease performance?(convert from float to int and CMP two ints, or just CMP two float values)